intranethome/views.py

```diff
@@ -0,0 +1,342 @@
+        if alias[0] in "0123456789":
+            list_of_numbers.append(alias)
+
+    for number in list_of_numbers:
+        all_aliases.remove(number)
+        all_aliases.append(number)
+
+    return all_aliases
+
+
+def get_list_of_sorted_aliases(data_from_file):
+    """
+    The function that selects for the alias values and sorts them while ignoring capitalization
+
+    Args:
+       data_from_file: all of the data that is pulled from the json file and converted to python
+
+    Returns:
+       a list of the fully ordered aliases after further being sorted by the helper function above
+    """
+
+    aliases = list({k for (k, v) in data_from_file.items()})
+    aliases.sort(key=str.lower)
+
+    aliases = helper_function_order_number_aliases_last(aliases)
+
+    return aliases
+
+
+def filter_by_value(original_alias_list, filter_value):
+    """
+    Filters the complete list of aliases by letter or number based on user input
+
+    Args:
+       original_alias_list: the full value of aliases that will filtered from
+
+       filter_value: the value that is being used as the filter (e.g. a letter like 'c', 'number' for all numerical aliases or '' to display all)
+
+    Returns:
+       a list of the filtered aliases
+    """
+
+    filtered_alias_list = []
+
+    for alias in original_alias_list:
+
+        # if the filter is number
+        if filter_value == "number":
+            if alias[0] in "0123456789":
+                filtered_alias_list.append(alias)
+
+        # if the filter is a single letter
+        elif filter_value != "":
+            if alias[0] == filter_value:
+                filtered_alias_list.append(alias)
+
+        # if the filter is nothing, indicating that all values should be displayed
+        else:
+            return original_alias_list
+
+    return filtered_alias_list
+
+
+def uniforming_into_list_of_dict(notes_or_emails):
+    """
+    Converts all of the emails and notes into the same format (lists of dictionaries) in order to be able to work with the data more easiily
+
+    Args:
+       notes_or_emails: a single note or email dictionary (option 1 or 3) or a list of notes and emails (option 2) that will be converted into the correct format
+
+        # option 1: single key dictionary with key = 'email' and value with a local or gobal email
+        #   {'email': 'postmaster'}
+
+        # option 2: list of single dictionaries with  key = 'email' and value with a single local or global email, in one of 3 formats
+        #   [{'email': 'lhauglan@midway.uchicago.edu'}, ..., {'email': 'd-larsen@uchicago.edu'}]
+        #   [{'email': 'fierrom@uchicago.edu (Maria Fierro)'}, ..., {'email': 'hartj@uchicago.edu (Jenny Hart)'}]
+        #   [{'email': 'Ariel Erbacher\t\t<aaescotese@uchicago.edu>'}, ..., {'email': 'Amy Boucher\t\t<aebouch@emory.edu>'}]
+
+        # option 3: dictionary with key = 'note'
+        #   {'note': 'emails go to command: "|/usr/local/mailman/mail/mailman post access-ip'}
```

Comment:
This is more clear for sure. I don't think the # symbols are adding anything though. I would remove those. It might be more clear if you added something like, "Possible input looks like this:" (or something to that effect, after "converted into the correct format". I really like how it says "(option 1 or 3)" and (option 2)" in the `notes_or_emails` argument description though, don't change that. That is very helpful!